src/lib/pipeline.ml

```diff
@@ -358,10 +368,17 @@ module Full (Bfx: Extended_edsl.Semantics) = struct
             ?rna:fastqc_rna email_options in
         Some [flagstat_email; fastqc_email] in
     let report =
+      let rna_bam, optitype_rna, stringtie, seq2hla, rna_bam_flagstat =
+        match rna_results with
+        | None -> None, None, None, None, None
+        | Some {rna_bam; optitype_rna; stringtie; seq2hla; rna_bam_flagstat} ->
+          Some rna_bam, optitype_rna, Some stringtie,
+          seq2hla, Some rna_bam_flagstat
+      in
       Bfx.report
         (Parameters.construct_run_name parameters)
         ?igv_url_server_prefix:parameters.igv_url_server_prefix
-        ~vcfs:maybe_annotated ?bedfile
+        ~vcfs:maybe_annotated (* ?bedfile *)
```

Comment:
is this a debugging artifact or uncommented on purpose? If the latter, might worth mentioning why ;)